### PR TITLE
File 212

### PR DESCRIPTION
**Pull request by JTY009**
Corrected and Updated README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -139,9 +156,8 @@ POST    /file-routing/requests
 | --------|---------|-------|
 | Created  | 201   | Successfully created event CLOSED |
```
Comment (konradwudkowski) [inline README.md]: it's not an event that is created

File: README.md [hunk 2]
```diff
@@ -139,9 +156,8 @@ POST    /file-routing/requests
...
 | Bad Request  | 400   |  Invalid Request. |
```
Comment (konradwudkowski) [inline README.md]: 400 = invalid request, this description doesn't add value, explaining when we may return 400 does

File: README.md [hunk 3]
```diff
@@ -251,7 +263,6 @@ GET     /file-transfer/envelopes/{envelope-id}
 | Ok  | 200   | File Successfully uploaded.  |
```
Comment (konradwudkowski) [inline README.md]: This section is all wrong... Where did you get 'file successfully uploaded' when this is download end-point? Where did you get 206 partially downloaded? Are we returning sth like that?

File: README.md [hunk 4]
```diff
@@ -37,7 +37,12 @@ POST   	/file-upload/envelopes
 
 Request (POST): localhost:8898/file-upload/envelopes
 
-Body: {}
+Body:
+``` json
+{
+    "callbackUrl": "{user provides}"
+}
```
Comment (konradwudkowski) [inline README.md]: is this the only possible param available? Is it optional?

Comment (JTY009) [inline README.md]: Should be optional.

**Comment by konradwudkowski** [comment]
I looked briefly and these things could be improved:

**Create Envelope**
why 2 example payloads if values in the second one are optional?

**Show Envelope**
"Show Envelope which comprises of Envelope Id and the current status." - is that all?

**Download file response and Download zip**
response is "download file" - should be something like "binary file" or "binary file containing zipped files"

**Create File Routing Request**
"400 Env not found" - is this correct?
also missing info on location

**Transfer**
Missing "destination" query string param - which is the most important here here!

**Pull request closed without merging**